geoips/commandline/ancillary_info/cmd_instructions.yaml

```diff
@@ -204,23 +215,49 @@ instructions:
       - Unit Test Name
   geoips_run:
     help_str: |
-      Run a geoips process workflow (procflow), given a GeoIPS package and a script
-      name. This will do the exact same thing as being in the directory of a certain
-      script, and running it via `./<script_name>`. To see what scripts are available,
-      you can run `geoips list-scripts <-p>`, where `-p` can be used alongside a GeoIPS
-      Package name if you want scripts from a single package.
+      Run a specified geoips process workflow (procflow) plugin. See
```

Comment:
I think this is technically a "processing workflow"?  I don't think it's worth updating, just a note.

To be fair, the only reference to what procflow means I found in docs is https://github.com/NRLMMD-GEOIPS/geoips/blob/main/docs/source/devguide/software_requirements_specification.rst buried in a sub bullet.